src/settingsmanager.cpp

```diff
@@ -18,14 +18,62 @@ SettingsManagerView* SettingsManager::getView()
     return view;
 }
 
+void SettingsManager::setSettings(const QString &key, const QVariant &value)
+{
+    m_settings.setValue(key, value);
+}
+
+void SettingsManager::deleteSettings(const QString &key)
+{
+    m_settings.remove(key);
+}
+
+bool SettingsManager::settingsExists(const QString &key)
+{
+    return m_settings.contains(key);
+}
+
+QVariant SettingsManager::getSettings(const QString &key)
+{
+    return m_settings.value(key);
+}
+
+float SettingsManager::getZoomFactorByZimId(const QString &id)
+{
+    qInfo() << "settings file path : " << m_settings.fileName();
```

Comment:
Do you still want so many lines of debug? Do we have a system to handle different level of log messages?

Reply:
It was temporary debug, I thought I will remove the commit once the problem is fixed. There is no options for different level of log message I think